### PR TITLE
fix: Remove Google from doc search action

### PR DESCRIPTION
**Pull request by richsalz**
Remove google from the "search lists" pulldown on the document page.

### CODE REVIEW

**Comment by rjsparks** [comment]
I think we should address what Rich points to above before (or instead of) merging this.

**Comment by rjsparks** [comment]
Just to make it clearer what the state of this is, I'm going to leave it open and request changes, but I'm not asking @richsalz for anything right now - wait for some design thinking please.

**Review by rjsparks — CHANGES_REQUESTED** [review]
Lets hold this for future conversation.

**Comment by richsalz** [comment]
@rjsparks please re-open; i removed the branch (which closed the PR), and then restored it.  oops.  Want to keep the commentary.  Or I can add the info here as notes to #6590 if that's better.

**Comment by rjsparks** [comment]
The commentary won't go away if you close the PR - we'll still be able to navigate to it from the issue.

**Comment by jennifer-richards** [comment]
Gah, adding to the complexity here I just accidentally closed this when I meant to close the window... sorry!

**Comment by rjsparks** [comment]
Closing this again - see #7009

**Pull request closed without merging**
